ema_workbench/em_framework/callbacks.py

```diff
@@ -18,6 +18,7 @@
 import shutil
 
 import numpy as np
+import numpy.ma as ma
```

Comment:
I wouldn't import `numpy.ma` separately, but just use `np.ma`. ma isn't a wide-used shorthand convention, so using plainly `.ma` adds a bit of ambiguity.

Reply:
I followed the convention as used on the numpy docs: https://numpy.org/doc/stable/reference/maskedarray.generic.html

Reply:
Ah right I see. For a NumPy heavy library that would be logical.

For us I think it's clearer if it's `np.ma` instead of `ma`, since then you immediately recognize it as an NumPy class.

Reply:
ok, I have modified it